mobile/screens/User/UserDashboardScreen.tsx

```diff
@@ -198,6 +235,54 @@ export default function UserDashboardScreen(props: any) {
 }
 
 const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: 22,
+  },
+
+  modalView: {
+    margin: 20,
+    backgroundColor: 'white',
+    borderRadius: 20,
+    padding: 35,
+    alignItems: 'center',
+    shadowColor: '#000',
```

Comment:
A blue shadow would probably be cool here, along with a blue border so it doesn't blend into the dashboard background as much